src/MoBi.Presentation/Mappers/SelectedIndividualToIndividualSelectionDTOMapper.cs

```diff
@@ -18,13 +18,12 @@ public SelectedIndividualToIndividualSelectionDTOMapper(IMoBiContext moBiContext
       {
          _moBiContext = moBiContext;
       }
-      
+
       public IndividualSelectionDTO MapFrom(IndividualBuildingBlock individualBuildingBlock)
       {
-         var individualSelectionDTO = new IndividualSelectionDTO(_moBiContext.CurrentProject.IndividualsCollection)
-         {
-            SelectedIndividualBuildingBlock = individualBuildingBlock
-         };
+         var individualSelectionDTO = new IndividualSelectionDTO(_moBiContext.CurrentProject.IndividualsCollection);
```

Comment:
we have an interface to retrieve the project. It's almost the same but abstract the fact that you are using the context all the time. Should we use this instead or it does nto make sense? IProjectRetriever or sthg like this. The context is the heavy duty object that does it all